docs/get-started/create-simple-transaction.md

```diff
@@ -29,7 +29,7 @@ cardano-cli query protocol-parameters \
 
 ```sh
 cardano-cli query utxo \
-    --address $(cat payment.addr) \
+    --address $(< payment.addr) \
```

Comment:
```suggestion
    --address $(cat payment.addr) \
```
The Dev Portal has always been agnostic about syntax preferences like this.  You'd have to back a long time to find a functional difference between these two constructions, but eventually the `cat` clause would be considered more robust because the null command would fail in other shells (e.g. `csh` which I used myself until several years ago).

Because so many of the edits to the prior documentation are about this single change, to consider your PR properly I believe all these subjective changes from `cat` to `<` everywhere have to be reverted.  We will then have a smaller, proper list of files to correlate with the new material and navigational changes.

If it still bothers you and/or associates I would recommend submitting an issue over a "`cat` vs. `<`" documentation standard which might then lead to a PR to change it separately.

Reply:
No problem,  I can revert changes to docs that are not mine. In the case of CLI and NODE stuff we will use $(< ) over cat. The main reason being shellcheck typically complains about the use of cat in this way.

Reply:
Given https://github.com/cardano-foundation/developer-portal/pull/1286#issuecomment-2264285287 (your volunteering to maintain these pages) we could leave this to your discretion then.  I won't let your choice of one or the other block my own approval of this material in any case... though I do think this PR reads more easily now that those particular changes have been reverted.